src/server/req_signal.c

```diff
@@ -178,7 +180,7 @@ req_signaljob(struct batch_request *preq)
 			if (get_subjob_state(parent, i) == JOB_STATE_RUNNING) {
 				pjob = find_job(mk_subjob_id(parent, i));
 				if (pjob) {
-					/* if suspending,  skip those already suspened,  */
+					/* if suspending,  skip those already suspended,  */
 					if ((susrsm == 1) &&
 						(pjob->ji_qs.ji_svrflags & JOB_SVFLG_Suspend))
```

Comment:
I'm not sure I'm following.  Both admin_suspend and suspend will suspend a job just the same.  The only difference is what happens to the state of the node.